### ISSUE TITLE
[1] Confirm the property: is_derived

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/louisville-hyku#148]
# Summary
in [file_set.rb](https://gitlab.com/notch8/louisville-hyku/-/blob/main/app/models/file_set.rb#L6) we updated the url for the is_derived property. Before, it was using an NNP reference. 
We need to confirm that it has been updated appropriately, or if it should be changed to: https://id.loc.gov/ontologies/bibframe.html#p_derivativeOf (or another value)

This ticket is more of a spike ticket/reminder vs something that is testable

**Comment by orangewolf** [comment]
Linked Items
---
[EPIC 1 - BULKRAX && TESTING](https://gitlab.com/notch8/louisville-hyku/-/issues/25)

[[3] file set behavior](https://gitlab.com/notch8/louisville-hyku/-/issues/45)

**Comment by orangewolf** [comment]
Alisha January 2022
---
`:is_derived` is not a property that exists in hyrax 2.9.6 (the version lv is currently on) or 2.9.5 (the version that nnp is currently on). "app/models/file_set.rb" is also not a file in either one of those versions. 

trying to confirm if this file/property is necessary at all to the work we're doing with bulkrax/ocr.

**Comment by orangewolf** [comment]
Alisha January 2022
---
full slack convo with lea ann: https://notch8.slack.com/archives/CFRN4PMMF/p1643408072364779

TLDR; we need `is_derived` so that the individual pages of a pdf will be created as jpg's and viewable to a signed in admin user. as a non admin or logged out user, the only item displayed should be the pdf. 
presently we can see the pdf and jpg's signed in as admin, but I see neither when logged out. I'll create a new ticket for that.

the link that's in the "predicate" part of this property isn't that important, but we should still change it from referencing nnp.

**Comment by orangewolf** [comment]
Summer January 2022
---
I think the link was already changed from referencing NNP in this commit by Shana last week: https://gitlab.com/notch8/louisville-hyku/-/commit/442bcf948798876f8643b8e058e70fac3b429a5e

It looks like it was changed from `https://nnp.wustl.edu/terms/isDerived` to `https://id.loc.gov/ontologies/bibframe.html#p_derivedFrom`so now from what I can tell it is not referencing NNP anymore.

**Comment by orangewolf** [comment]
Alisha January 2022
---
new ticket to complete the work related to file sets: [#45 (closed)](https://gitlab.com/notch8/louisville-hyku/-/issues/45)

**Comment by orangewolf** [comment]
Shana January 2022
---
@alishaevn I talked to Lea Ann about this and she said that the link I was concerned about is actually not important in terms of functionality. (not to say that we shouldn't change it) but just adding context if it helps. ie, we could put google.com there and it should still work.[@alishaevn](https://github.com/alishaevn) I talked to Lea Ann about this and she said that the link I was concerned about is actually not important in terms of functionality. (not to say that we shouldn't change it) but just adding context if it helps. ie, we could put google.com there and it should still work.
